Headers/wayland/WaylandServer.h

```diff
@@ -132,7 +139,7 @@ struct window {
     struct xdg_positioner *positioner;
     struct zwlr_layer_surface_v1 *layer_surface;
     struct output *output;
-    WaylandCairoSurface *wcs;
+    _CAIRO_SURFACE_CLASSNAME *wcs;
```

Comment:
Could this variable just be a CairoSurface? I don't see any specific method you are using on it after the creation. But I might be wrong here as the PR only shows the changed lines.

Reply:
good suggestion,
I closed this PR and made a new one, rebasing the code on the wayland branch. this change is applied there